### PR TITLE
Html css styling

### PR DESCRIPTION
**Pull request by samscupoftea**

### CODE REVIEW

**Review by josueJURE — COMMENTED** [review]
Hi, for some reason I can't see the changes you've made. I'm afraid. By the way do we have a working API key?

**Comment by samscupoftea** [comment]
You can't see them? Have you clicked view reviewed changes button here https://github.com/fac29b/recipe-for-success/pull/4/files
Also try pulling the project to your vs code. I had to adapt write html and css as you didn't make any styling for this project, just the react one. So I did my best to adapt that version to the broken up structure of the components. I think we have the backup API key we can use. I didn't push that.

File: public/index.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
+:root {
```
Comment (samscupoftea) [inline public/index.css:1]: Some of this is copied CSS from Josue's component CSS files from our react project

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```
Comment (samscupoftea) [inline public/index.html:1]: Basic HTML structure similar to React version.

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <title>Recipe For Success</title>
+  </head>
+
+  <body>
+    <main aria-label="container" class="container">
```
Comment (samscupoftea) [inline public/index.html:12]: Include a main div for container to for universal styling.

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      <h1>
+        Don't know what to cook? No worries recipe for success app does the
+        thinking for you.
+      </h1>
+      <!-- Placeholder for Input component -->
+      <div class="input-placeholder">Input Component Placeholder</div>
```
Comment (samscupoftea) [inline public/index.html:18]: Input placeholder is for whatever input we decide to place here.

File: public/index.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      <section class="button-section">
+        <!-- Placeholder for Button components -->
+        <button>Italian</button>
+        <button>Chinese</button>
+        <button>Peruvian</button>
+        <button>Surprise me</button>
+      </section>
+      <!-- Placeholder for FetchOpenAI component -->
+      <div class="fetch-openai-placeholder">
```
Comment (samscupoftea) [inline public/index.html:27]: This is where the open AI response will go.

File: server.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+const fetch = require('node-fetch');
+require('dotenv').config();
+const {configuration, OpenAI } = require ('openai');
+const app = express();
```
Comment (samscupoftea) [inline server.js:5]: All the relevant require's needed to enable the page to work. 
Make sure to install express, node fetch, dot env and open ai.

File: server.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+    });
+    res.json(completion);
+     console.log(completion);
+});
+
+// app.get('/', (req, res)=> {
+//     res.send('welcome to my server!');
+// });
+
+app.use(express.static('Public'));
```
Comment (samscupoftea) [inline server.js:30]: Hosted on port 3000

File: server.js [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+const fetch = require('node-fetch');
+require('dotenv').config();
+const {configuration, OpenAI } = require ('openai');
+const app = express();
+
+// const openai = new OpenAI({
+//     apiKey: process.env.openaiAPI,
+// });
+
+app.get("/openai", async (req, res) => {
```
Comment (samscupoftea) [inline server.js:11]: Basic open AI get request. Needs expanding upon.

**Review by samscupoftea — COMMENTED** [review]
In summary, this is just a rough and basic outline structure. I am now working on server.js and front end JS for open ai integration

**Pull request merged**